cmd/get.go

```diff
@@ -36,23 +62,33 @@ This command is useful for creating and updating Release notes in GitHub.
 	RunE: func(command *cobra.Command, args []string) error {
 		fileName := configuration.Config.FileName
 
+		var tmplSrc string
 		var changelog changelog.Changelog
 		var err error
 
 		if printLatest {
 			changelog, err = get.GetLatest(fileName)
 		} else if printVersion != "" {
 			changelog, err = get.GetVersion(fileName, printVersion)
+		} else if outputTemplate == outputNotes {
```

Comment:
For this comparison, we can just do a simple string one:

```suggestion
		} else if outputType == "notes" {
```

Reply:
outputType makes sense, but comparing to a string does not. All of this ties back to command flag validation and using constants to expose issues early, at compile time.